qoqo/src/operations/mod.rs

```diff
@@ -148,5 +152,9 @@ pub fn operations(_py: Python, m: &PyModule) -> PyResult<()> {
     m.add_class::<EchoCrossResonanceWrapper>()?;
     m.add_class::<PragmaAnnotatedOpWrapper>()?;
     // 1.9
+    // 1.10
+    m.add_class::<QuantumRabiWrapper>()?;
+    m.add_class::<LongitudinalCouplingWrapper>()?;
+    m.add_class::<JaynesCummingsWrapper>()?;
```

Comment:
You'll probably need to add the unstable feature here too

Reply:
Fixed